syntax = "proto3";

package xray.common.serial;
option go_package = "github.com/xtls/xray-core/common/serial";

message TypedMessage {
  string type = 1;
  bytes value = 2;
}
